src/util/endpoints.js

```diff
@@ -58,10 +60,22 @@ const fetchEndpoints = async (fsp) => {
   ).startTimer()
   try {
     Logger.isDebugEnabled && Logger.debug(`[fsp=${fsp}] ~ participantEndpointCache::fetchEndpoints := Refreshing the cache for FSP: ${fsp}`)
-    const defaultHeaders = Http.SwitchDefaultHeaders(Enum.Http.HeaderResources.SWITCH, Enum.Http.HeaderResources.PARTICIPANTS, Enum.Http.HeaderResources.SWITCH)
+    if (!hubName) {
+      throw Error('"hubName" is not initialized. Initialize the cache first.')
+    }
+    if (!hubNameRegex) {
```

Comment:
Can't we just calculate `hubNameRegex` here based on `hubName` (to avoid making it required)?, e.g.:
```
const { HeaderValidation } = require('@mojaloop/central-services-shared').Util
...
const hubNameRegex = HeaderValidation.getHubNameRegex(Config.HUB_NAME)
```

Reply:
You're right. Would reduce the number of parameters there.